### DISCUSSION TITLE
Default TMDB API key for the project

### DISCUSSION THREAD

**Discussion by cgomesu** [jsaddiction/TrailerTech#17] [Ideas]
@jsaddiction , thanks for taking the time to write this program.  I've started testing it and noticed that the project does not have a default TMDB API key--users are instructed to create an account and then request a key themselves.  According to [TMDB's API FAQ](https://www.themoviedb.org/faq/api), there is no rate limit on the API and keys are free as long as TMDB is credited as the data source.  This default API key could be added to the `settings.ini` file and if any user wants to use their own, they can edit the file accordingly.  Maybe you know of a limitation to this but I've seen multiple projects using a default key without any issues.  What are your thoughts on this?

**Comment by jsaddiction** [comment]
I have read through TMDB's site and created its own hardcoded API key.  The thought originally was if TMDB decided one day to drop support for unlimited api calls then all users would be effected.  

After looking through the documentation and considering the above, I have decided to implement a hybrid solution.  As long as the api_key field in settings.ini is left blank a blanket API key is used.  If for any reason you see a need any text in that setting overrides the default.

Great Idea!
